src/schema/me/followed_fairs.ts

```diff
@@ -0,0 +1,62 @@
+import { ProfileType } from "schema/profile"
+import Fair from "schema/fair"
+import { IDFields } from "schema/object_identification"
+
+import { pageable, getPagingParameters } from "relay-cursor-paging"
+import { connectionDefinitions, connectionFromArraySlice } from "graphql-relay"
+import { GraphQLObjectType } from "graphql"
+
+const FollowedProfileEdge = new GraphQLObjectType({
+  name: "FollowedProfileEdge",
+  fields: {
+    profile: {
+      type: ProfileType,
+    },
+    ...IDFields,
+  },
+})
+
+// the challenge is that FollowedProfile doesn't directly connect to Fair.
+// we need to go from FollowedProfile to Profile to Owner in order to get the
+// reference to the fair object - how do I tell graphQL how to do that?
+// Is it just manually down in my resolver below?
+export const FollowedFairConnection = connectionDefinitions({
+  name: "FollowedFairConnection",
```

Comment:
Wondering if this should contain "Connection" in the name? We are getting redundant "...ConnectionConnection" naming [up here](https://github.com/artsy/metaphysics/pull/1441/files#diff-ca178747ca9568ddd30e7dffd824b873R3547), presumably because of [the way the library handles this](https://github.com/graphql/graphql-relay-js/blob/88a2c88f53634cc7cae7ff56e33f678b9f34b7d5/src/connection/connection.js#L111).

[Ditto](https://github.com/graphql/graphql-relay-js/blob/88a2c88f53634cc7cae7ff56e33f678b9f34b7d5/src/connection/connection.js#L93) for "Edge" in [L10](https://github.com/artsy/metaphysics/pull/1441/files#diff-908679437786a6422f3da2c394b44d14R10) above.

Reply:
Good catch, made the change! I've seen different approaches for Edges and unless someone specifically finds this one problematic I think I'm okay with it but also 100% open to any feedback!